cypress/integration/SEP.ts

```diff
@@ -137,6 +139,7 @@ function createWorkflowAndEsiTemplate() {
 }
 
 function initializationBeforeTests() {
+  cy.getAndStoreFeaturesEnabled();
```

Comment:
Do you think we could move `cy.getAndStoreFeaturesEnabled();` into cypress/support/beforeRunHook.ts so that it is run always before every test, and we can clean up these before hooks in each test?

It does not hurt if it is not used, right?

Reply:
Just putting into the cypress/support/beforeRunHook.ts doesn't seem to work as it doesn't get the features enabled and just returns an empty object. I think `cy.getAndStoreFeaturesEnabled()`; needs to be in each of the beforeEach.

Reply:
Yes, I see that the hook is run before the setup of the DB.